CMakeLists.txt

```diff
@@ -411,6 +416,23 @@ endif()
 
 # CXX Compiler settings
 set(CMAKE_CXX_STANDARD 14)
+
+if(OPENMW_LTO_BUILD)
+    if(POLICY CMP0069)
+        include(CheckIPOSupported)
+        check_ipo_supported(RESULT HAVE_IPO OUTPUT HAVE_IPO_OUTPUT)
+        if(HAVE_IPO)
+            message(STATUS "LTO enabled")
+            set(CMAKE_INTERPROCEDURAL_OPTIMIZATION TRUE)
+        else()
+            message(WARNING "Requested option OPENMW_LTO_BUILD not supported by this compiler: ${HAVE_IPO_OUTPUT}")
+        endif()
+    else()
+        message(WARNING "Requested option OPENMW_LTO_BUILD not supported by this cmake version: ${CMAKE_VERSION}")
```

Comment:
I'm pretty sure it *will* be supported, just not necessarily for the current compiler.

Reply:
I am not sure how much work we should put into supporting:
icc users on linux with cmake < 3.9

Reply:
Now I'm aware of how bad the setting used to be, I see your point. I don't necessarily agree that it's a good idea to do it quite like this. It's probably better to log that it *probably* isn't supported, and try setting it anyway.

Reply:
E.g. if someone's stuck on a distro with ancient CMake, they could provide their own toolchain file for their tools, and include the LTO flags in that, and CMake would suddenly be able to set them.